code/datums/jobs/access.dm

```diff
@@ -343,5 +343,11 @@
 			. = size ? "MERC " : "MERC Engineer"
 		if("VM")
 			. = size ? "VAT " : "VatGrown Marine"
+		if("CS")
+			. = size ? "CS " : "Chief Synthetic"
+		if("MS")
+			. = size ? "MS" : "Major Synthetic"
+		if("SO1")
+			. = size ? "SO1 " : "Synthetic First Class"
```

Comment:
synthetic having military ranks is hella weird. I prefere the MK thing

Reply:
Yea old non updated branch.